pkgs/npm/src/installer.ts

```diff
@@ -41,7 +41,7 @@ async function downloadFile(
     fs.writeFileSync(downloadTargetFile, response.data);
```

Comment:
nit: if you're already in an `async` function, use the asynchronous functions from `"fs/promises"` package instead: `await fs.unlink(…)` etc.